.travis.yml

```diff
@@ -1,19 +1,10 @@
-language: erlang
+language: elixir
+elixir:
+  - 0.15.1
```

Comment:
This doesn't seem to work in travis. But support for 0.15.1 is not really needed anymore. Can you just delete this line? Then I'll merge it.